### DISCUSSION TITLE
How to make the debugger UI split into four quadrants?

### DISCUSSION THREAD

**Discussion by aj3423** [foundry-rs/foundry#8250] [Q&A]
The debugger layout in the [document](https://book.getfoundry.sh/forge/debugger#debugger-layout) looks like:
![image](https://github.com/foundry-rs/foundry/assets/4710875/848edc34-3cbc-4155-810a-197fe58f0b9c)

But mine actually looks like:
![image](https://github.com/foundry-rs/foundry/assets/4710875/c3f5f652-3c9d-4c74-b551-f2128f34499b)

How to change the layout to four quadrants?

Foundry version:  forge 0.2.0 (ba9fa20 2024-06-24T00:20:52.822818467Z)
OS: Fedora 40, KDE

**Comment by mds1** [comment]
This is based on window size, if you expand your terminal window to be bigger you'll see the top view

**Comment by aj3423** [comment]
My display resolution is 1920x1080, forge debugger window is maximized but the layout still doesn't change. Does it need to be larger than 1920x1080? I tried both Wezterm and Kitty terminal, no difference.

![image](https://github.com/foundry-rs/foundry/assets/4710875/2be16fba-d343-4072-9ef8-4a6367790baf)

**Reply by mds1** [reply]
I am not sure, I would suggest creating an issue if it's not working for you

**Reply by aj3423** [reply]
Thanks, I fired an issue: https://github.com/foundry-rs/foundry/issues/8256, closing this.
